ReactCommon/cxxreact/Instance.h

```diff
@@ -82,7 +67,6 @@ class RN_EXPORT Instance {
   void callJSFunction(std::string &&module, std::string &&method,
                       folly::dynamic &&params);
   void callJSCallback(uint64_t callbackId, folly::dynamic &&params);
-  virtual void setJSEConfigParams(std::shared_ptr<JSEConfigParams>&& jseConfigParams);
```

Comment:
>setJSEConfigParams [](start = 15, length = 18)

This is called from ReactNativeHost on Office on Android ..

Reply:
This will be removed from Repo with my PR.. Could you wait this PR until mine is checked in ? Doing some final validations now ..

---
In reply to: [390611959](https://github.com/microsoft/react-native/pull/257#discussion_r390611959) [](ancestors = 390611959)

Reply:
Oh wow, you're right. I'm not sure how I missed that.

I can wait and recreate this after your change.